lib/bolt/cli.rb

```diff
@@ -175,6 +175,7 @@ def load_config
     # Completes the setup process by configuring Bolt and log messages
     def finalize_setup
       Bolt::Logger.configure(config.log, config.color, config.disable_warnings)
+      Bolt::Logger.stream = options[:stream] || config.stream
```

Comment:
Yeah, I think that `configure` is more meant to configure the Logger itself, whereas this variable is only attached to the logger so that's it's available basically globally (and because it's vaguely logger-related). I actually originally had it as a var to `configure` then pulled it out.